tests/integration/ha/continuous_writes.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+import subprocess
+import sys
+import time
+
+from literals import SNAP_NAME
+
+logger = logging.getLogger(__name__)
+
+
+def continuous_writes(model: str, unit: str, endpoints: str, user: str, password: str):
```

Comment:
side note for a next PR, as discussed, please make use of the OpenSearch continuous writes structure - it's more robust and flexible.